db/process_message.c

```diff
@@ -605,6 +605,9 @@ void bdb_newsi_mempool_stat();
 static pthread_mutex_t exiting_lock = PTHREAD_MUTEX_INITIALIZER;
 void *clean_exit_thd(void *unused)
 {
+    if (thedb->bdb_env == NULL)
```

Comment:
check gbl_ready instead/also?

Reply:
Thanks for the suggestion. Checking gbl_ready is indeed better here.